docs/events/wg_workshops/2022-12-05-december-meeting/index.md

```diff
@@ -14,4 +14,274 @@ breakout-tre-standardisation.md
+
+It was agreed that the solutions vary and depend on both risk apetite and trustiness of packages (or rather of the communities developing and maintaning them). 
+
+CHAT Qs/reflections:
+- The dependency management issue is very much similar to that faced by HPC facilities. This adds an additional security challenge but the problem is familiar to HPC centre manager teams. {module files, spack, easy build etc}
+- How hard/ time consuming is ^ the way HPC do things, would it be practical to copy their approach each time a cloud deployable TRE is set up?
+- Whitelisting suggests making sure the software is compliant and safe before using it. Other processes can be used to verify nothing wrong happened in the environment AFTER the software was used. Cost of prevention vs. cost of remediation.
+- . Is there a difference between interpreted vs compiled code in terms of TRE safety/risk?
+
+**Actions/next steps**
+
+* Possiblity for a shared list of 'trusted' software packages as part of this community
+
+[**Full notes**]()
+
+### Session 2
+
+#### **Cloud TREs: How do we fairly charge researchers?**
+
+Simon Li (HIC)
+
+[name=Hari] Any summary???
+
+Participants reflected on the balance between what can be offered for free and what should entail additional costs. There was a certain agreement on a level of offering not requiring to be charged but not so much yet as to where that stands.
+
+Part of the difficulty in projects being charged lies in researchers not fully understanding the technical cost (both in hardware and developer time). An effective way of mitigating this is demoing researchers to show impacts on performance rather than listing computing specifications. Another part has been when charging for storage that if you do users have used their own storage under their desk but providing it free has meant projects not applying for grants to cover TREs costs.
+
+Another thing to take into account is the different service model of AWS and Azure, the latter closing GPUs down when credits are consumed while AWS allows for over-expenditure.
+
+While there are still no clear answers it looks like certain standarization across proejcts is needed, as well as a way to ensure at least some central funding for TREs at institutional and project levels.
+
+
+
+[**Full notes**]()
+
+#### **User experience in the Azure OSS TRE** 
+
+Alessandro Riccombeni (Microsoft)
+
+From Microsoft leads. tells a bout the UX in their TREs and the functionalities that an user need. A key Challenge as user of a TRE? What would solving the challenge unlock?
+ ![](https://i.imgur.com/Gf8xjBz.png)
```

Comment:
Would you mind copying this image into this repo in case the source is deleted?

Reply:
I see it already saved? downleaded just in case and can add where necessary